packages/core/schema/index.graphql

```diff
@@ -944,6 +944,15 @@ type Customer {
   addresses: [CustomerAddress!]!
 }
 
+type AuthResult {
+  id: ID!
+  authToken: String
+  refreshToken: String
+  expiresIn: String
+  customer: Customer
+  errors: [Error]
```

Comment:
Would it make sense to simply throw an error in case of an unsuccessful authentication attempt? We then could omit the `errors` field make the customer field required: `customer: Customer!`